### ISSUE TITLE
chuangmi_ipc019 Camera gives notification every 5 min when plugin is enabled in home assistant

### ISSUE THREAD

**Issue by milindpatel63** [al-one/hass-xiaomi-miot#622]
I added this plugin in home assistant and I am using it to control my Mi Home 360 Camera (chuangmi_ipc019).

But the camera has 3 entities,
1. A binary switch to turn off or on
2. A camera control entity which gives live stream
3. A motion entity which shows last motion detection capture video.

The problem is,
This plugin doesn't work unless I have the live stream entity enabled.
If I disable the live stream entity, all other entities become unavailable.

But, if I enable the live stream entity, all the members of my family keep getting notifications in their Mi Home app that a user is watching live stream on a smart display.
This notification keeps coming even if i have not opened home assistant or added that live stream entity in my dashboard.

So basically, this plugin is not useful for now for "chuangmi_ipc019" camera if you also use Mi Home android/ios app.

**Comment by al-one** [comment]
Add `use_motion_stream: true` for this device.
<img width="391" alt="xiaomi miot camera" src="https://user-images.githubusercontent.com/4549099/170942859-dfb4a163-575f-4382-9c8d-df948bcace73.png">

**Comment by tbarbette** [comment]
Same problem here ! With use_motion_stream the stream does not work anymore. It only shows the last motion event.


PS: I'm on the right repo this time :)

**Comment by al-one** [comment]
Live streaming is not supported. It's my best effort to see the last motion event.

**Comment by milindpatel63** [comment]
> Add `use_motion_stream: true` for this device. <img alt="xiaomi miot camera" width="391" src="https://user-images.githubusercontent.com/4549099/170942859-dfb4a163-575f-4382-9c8d-df948bcace73.png">

Ok, I may just be dumb. But I don't have that customize entity UI anywhere in my home assistant.
How do I get to that setting?



> Live streaming is not supported. It's my best effort to see the last motion event.

Also, live streaming works for me for this camera. The feed is decent. Lags a bit sometimes, but after it starts, it's smooth.

**Comment by tbarbette** [comment]
> Live streaming is not supported. It's my best effort to see the last motion event.

Yes it actually works, it's only about that unsupportable notification :p

**Comment by milindpatel63** [comment]
I think people are getting confused here about the issue.

Let me clarify again.

I want,

With all 3 entities configured for this camera, NOT get notification in Mi Home app every 5 minutes saying that "A user is watching live stream on a smart display"

OR

Keep the 2 entities (live stream one and motion alert one) disabled in home assistant, so I don't get that notification, and still be able to use the third entity(the switch one) to turn on or off the camera.

I believe, there is no way to stop that notification, because this plugin uses Xiaomi API to access live stream, which then generates that notification, and it can only be turned off from Xiaomi's side.

So my solution would be to,
Allow the third entity (the switch one) to operate, while having other 2 disabled.

Right now, the third entity becomes unavailable in home assistant if I disable any of the other two.

**Comment by al-one** [comment]
`camera.xxxx_camera_control` is a parent entity and cannot be disabled. All states updates of the device are based on it. Try hiding the entity in HA.
<img width="391" alt="image" src="https://user-images.githubusercontent.com/4549099/171318146-718ca0e3-6698-4cd3-bc9a-fd8d49a78fe9.png">

**Comment by milindpatel63** [comment]
> `camera.xxxx_camera_control` is a parent entity and cannot be disabled. All states updates of the device are based on it. Try hiding the entity in HA.
> <img width="391" alt="image" src="https://user-images.githubusercontent.com/4549099/171318146-718ca0e3-6698-4cd3-bc9a-fd8d49a78fe9.png">
> 

Tried it. Hiding the entity still causes the plugin to poll the camera every 5 minutes or so, and this causes the "Mi home" app to generate a new notifications every time this plugin polls the camera.

**Comment by al-one** [comment]
> We could also only request the stream address when user clicks on the entity or it's card, rather than polling the stream address every few minutes. This might also help in reducing the frequency of that notification.

Make sure your `keep_streaming` option is turned off.

**Comment by milindpatel63** [comment]
> > We could also only request the stream address when user clicks on the entity or it's card, rather than polling the stream address every few minutes. This might also help in reducing the frequency of that notification.
> 
> Make sure your `keep_streaming` option is turned off.

Even after keeping the "keep_streaming" option off for all the entities, if I put a picture card on dashboard, it loads the screenshot image, and this causes the notification still....so I can't even keep the entity on dashboard without constant spamming notifications.... :(

**Comment by al-one** [comment]
Do not use `camera.xxxx_camera_control` as a card unless you add custom option `use_motion_stream: true` to it.
